### ISSUE TITLE
NullPointerException on app switch / screen lock

### ISSUE THREAD

**Issue by filalex77** [TeamNewPipe/NewPipe#1954]
Usually (like 90% chance) happens after locking the screen with playing video and unlocking back. Sometimes happens after switching apps through Recent apps and back.

Crash dump after unlocking the screen:

```
{
   "user_action": "ui error",
   "request": "App crash, UI failure",
   "content_language": "US",
   "service": "none",
   "package": "org.schabi.newpipe",
   "version": "0.14.2",
   "os": "Linux Android 5.1 - 22",
   "time": "2018-12-19 12:23",
   "exceptions": [
      "java.lang.NullPointerException: Attempt to invoke virtual method 'boolean com.google.android.exoplayer2.SimpleExoPlayer.getPlayWhenReady()' on a null object reference\n\tat org.schabi.newpipe.player.VideoPlayer.onStartTrackingTouch(VideoPlayer.java:713)\n\tat android.widget.SeekBar.onStartTrackingTouch(SeekBar.java:119)\n\tat android.widget.AbsSeekBar.onTouchEvent(AbsSeekBar.java:602)\n\tat android.view.View.dispatchTouchEvent(View.java:8668)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2523)\n\tat android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2114)\n\tat com.android.internal.policy.impl.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2539)\n\tat com.android.internal.policy.impl.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1807)\n\tat android.app.Activity.dispatchTouchEvent(Activity.java:2823)\n\tat android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)\n\tat com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2496)\n\tat android.view.View.dispatchPointerEvent(View.java:8874)\n\tat android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4729)\n\tat android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4587)\n\tat android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4086)\n\tat android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4139)\n\tat android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4105)\n\tat android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4242)\n\tat android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4113)\n\tat android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4299)\n\tat android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4086)\n\tat android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4139)\n\tat android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4105)\n\tat android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4113)\n\tat android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4086)\n\tat android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:6543)\n\tat android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:6517)\n\tat android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:6470)\n\tat android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6723)\n\tat android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:185)\n\tat android.os.MessageQueue.nativePollOnce(Native Method)\n\tat android.os.MessageQueue.next(MessageQueue.java:148)\n\tat android.os.Looper.loop(Looper.java:151)\n\tat android.app.ActivityThread.main(ActivityThread.java:5877)\n\tat java.lang.reflect.Method.invoke(Native Method)\n\tat java.lang.reflect.Method.invoke(Method.java:372)\n\tat com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1015)\n\tat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:810)\n\tat de.robv.android.xposed.XposedBridge.main(XposedBridge.java:107)\n"
   ]
}
```

**Comment by TobiGr** [comment]
Please go through the check-list. This is a duplicate and already fixed (in version 0.15.0) and do not paste crash logs.

**Comment by filalex77** [comment]
@TobiGr I didn't notice that, sorry. Thanks for the time!
